### ISSUE TITLE
soc estimation

### ISSUE THREAD

**Issue by mooncode30** [MarcBerliner/PETLION.jl#106]
can petlion used to find estimation soc lithium-ion while charging and discharging?

**Comment by MarcBerliner** [comment]
Yes, you can access the SOC and SOH by `sol.SOC` and `sol.SOH`. To enable SEI degradation, you must set `aging = true` when creating the model using `petlion()`.

**Comment by mooncode30** [comment]
hi sir, i wanna ask again.
i wanna know the reason why in graph both state of health and state of charge is (-) ?
![image](https://github.com/MarcBerliner/PETLION.jl/assets/132383791/faeff747-142d-4c19-b3dd-f57644dda14f)

**Comment by mooncode30** [comment]
so it can't be determined by percent? for example: charging when soc capacity is 80%

**Comment by MarcBerliner** [comment]
80% is an SOC of 0.8.

**Comment by mooncode30** [comment]
why in pict soc and soh is (-)?

**Comment by mooncode30** [comment]
dimension of LiB?
if with dimension can?
